test/e2e-tests.sh

```diff
@@ -37,9 +37,15 @@ function run_buildpack_test() {
   kubectl apply -f test/build-buildpack.yaml || return 1
   # Wait 5s for processing to start
```

Comment:
I believe that you can remove this sleep now, since you're checking for a successful build.